http-proxy/conf.d-prod/ngnix.conf

```diff
@@ -24,7 +24,7 @@ server {
     }
 
     location /tiler/ {
-        proxy_pass http://biab-tiler:8000/;
+        proxy_pass http://biab-python-api:8000/;
     }
 
     location /region/ {
```

Comment:
should this proxy-pass it to http://biab-python-api:8001/region/ [...], so that it's easier to extend the python API for other things than regions in the future?